README.md

```diff
@@ -46,12 +46,14 @@ You can find a demo of the project [here](https://github.com/reubenae/user-data-
 
 ## Contributing
 
+We do not allow contributors to claim issues. If you find something interesting you can contribute to the repo, feel free to raise a PR. We do not require you to let us know in advance. 
+
 1. Fork the repo
 1. Install dependencies locally `yarn` or `npm install`
 1. Make your changes.
 1. Make sure it builds `yarn build` or `npm build`
 1. Run the tests (you added tests, right?) with `npm test` or `yarn tests`
-1. Test your changes in your consuming code: `yarn link` [Yarn link documentation](https://classic.yarnpkg.com/en/docs/cli/link) or `npm link` [npm link documentation](https://docs.npmjs.com/cli/link)
+1. Test your changes in your consuming code: [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link) or [`npm link`](https://docs.npmjs.com/cli/link)
```

Comment:
Please change the text to :
Test your changes in your consuming code or using our demo project:

Reply:
Done